code-challenges.test.js

```diff
@@ -52,5 +111,25 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+describe("a funciton called accSum", () => {
+    it("takes in an array and returns an array of the accumulating sum", () => {
+        expect(accSum(numbersToAdd1)).toEqual([2, 6, 51, 60]);
+        expect(accSum(numbersToAdd2)).toEqual([0, 7, -1, 11]);
+        expect(accSum(numbersToAdd3)).toEqual([]);
+    })
+
 
+})   
 // b) Create the function that makes the test pass.
+//create a function that takes in an array
+//create an empty array to store the new values in
+//iterate over the array provided
+//add the first value to itself
+//add the sum of the first value and itself to the next value and so on
+//return the accumulating sum into the new array
+
+const accSum = (array) => {
+    let accArr = array.map((sum => value => sum += value)(0));
+    return accArr
+    }
```

Comment:
A general rule of thumb is if you are creating a function for the sole purpose of returning it, you can just return your statement.